docs/source/xrpl.asyncio.rst

```diff
@@ -1,22 +1,16 @@
-xrpl.asyncio package
-====================
+XRPL Async Features
+===================
 
-Subpackages
------------
+The `xrpl-py` library supports `Python's asyncio implementation <https://docs.python.org/3/library/asyncio.html>`_. All of these methods are analogous to the synchronous ones.
+
+If working with asynchronous code, **do not** use synchronous methods. Due to the way that loops are handled, this will not work. Each synchronous method has an analogous asynchronous method that should be used instead.
```

Comment:
it's not that dire. if you tried to do this python will warn you anyway. i think we should ease off on the tone, "if working with async code, you cannot use the synch methods and should instead use their async counterparts".